www/js/survey/enketo/enketoHelper.ts

```diff
@@ -266,6 +267,13 @@ export function saveResponse(
               : opts.timelineEntry.exit_ts,
           };
         }
+        // if timestamps were not resolved from the survey, we will use the trip or place timestamps
```

Comment:
Why is this needed? Can you provide context?

Reply:
The if/else block above should already handle this

Reply:
I think that was a bad merge conflict, Thank you for pointing it out! It's been so long since I first wrote this so I'm having a hard time cleaning it back up, moved it back to "in progress" until I can get the tests running again!